src/f32/vec3_mask.rs

```diff
@@ -31,6 +34,33 @@ impl Default for Vec3Mask {
     }
 }
 
+#[cfg(vec3sse2)]
+impl PartialEq for Vec3Mask {
+    fn eq(&self, other: &Self) -> bool {
+        let self_arr = unsafe { &*(self as *const Vec3Mask as *const [f32; 3]) };
+        let other_arr = unsafe { &*(other as *const Vec3Mask as *const [f32; 3]) };
+
+        self_arr
+            .iter()
+            .zip(other_arr.iter())
+            .all(|(a, b)| a.to_bits().eq(&b.to_bits()))
+    }
+}
+
+#[cfg(vec3sse2)]
+impl Eq for Vec3Mask {}
+
+#[cfg(vec3sse2)]
+impl hash::Hash for Vec3Mask {
+    fn hash<H: hash::Hasher>(&self, state: &mut H) {
+        let self_arr = unsafe { &*(self as *const Vec3Mask as *const [f32; 3]) };
```

Comment:
Could use `.into()` here if `impl From<Vec3Mask> for [u32; 3]` was added as I mentioned above.